Makefile.source.mjs

```diff
@@ -92,7 +92,7 @@ target["clean-all"] = function () {
     shell.rm("-rf", `${source}/*/package-lock.json`);
   });
 
-  target["clean"]();
+  target.clean();
```

Comment:
Could we disable the rule in this file? `target` is used more like a map, and I quite like that we are using string-keyed keys with it.

Reply:
Would you like to use `allowKeywords: false` for `babel-helpers` too?

Reply:
I personally prefer to keep `babel-helpers` as is.

Reply:
I think the current babel-helpers change are required for ES3 compatibility. We have such patterns in regenerator runtime as well, but it was not enforced consistently in the helpers.

Reply:
Maybe we can unify it to es6 in Babel 8, since it can be easily transformed by `preset-env`.
For example, `obj.return`, `{fn(){}}`.

Reply:
I think it is non-trivial to apply preset-env on babel-helpers, if it is an ES2015 build: there are quite a few manual polyfills, feature detections, etc. They are prone to errors if we task consumers to transform them. Maybe we can provide an ES2015 build and an ES3 build. If the `targets` support ES2015, the `transform-runtime` emits imports to the ES2015 build, otherwise we default to the ES3 build.

Reply:
I mean only use syntax that can be easily transformed. :)
Of course your suggestion seems better.